examples/flask/flask_seguro/cart.py

```diff
@@ -1,9 +1,15 @@
-from flask_seguro.products import Products
+""" The file is responsable for cart in flask-webpage """
+
 from flask import current_app as app
+from flask_seguro.products import Products
+
 
+class Cart(object):
+    """ The classe is responsable for cart in webpage """
 
-class Cart:
     def __init__(self, cart_dict=None):
+        """ Initializing class """
```

Comment:
I think we can clean the comments a little bit. We should avoid redundant comments and focus on the ones that are really needed (when we need to explain why, instead of how). Reference for a good [post](https://www.todaysoftmag.com/article/1120/clean-code-comments-and-formatting) about that based on the Clean Code book.

Reply:
Yes, i do. Sorry, but, if i did not do that, the tests is be failed

Reply:
MacOS files is ignored